README.md

```diff
@@ -1,25 +1,103 @@
-Adding your package to conda-forge
-----------------------------------
+## Build status
+
+[![Circle CI](https://circleci.com/gh/conda-forge/staged-recipes/tree/master.svg?style=shield)](https://circleci.com/gh/conda-forge/staged-recipes/tree/master)
+
+[![Build Status](https://travis-ci.org/conda-forge/staged-recipes.svg?branch=master)](https://travis-ci.org/conda-forge/staged-recipes)
+
+[![Build status](https://ci.appveyor.com/api/projects/status/3lju80dibkmowsj5/branch/master?svg=true)](https://ci.appveyor.com/project/conda-forge/staged-recipes/branch/master)
+
+## Getting started
 
 1. Fork this repository.
 2. Make a new folder in `recipes` for your package. Look at the example recipe and our [FAQ](https://github.com/conda-forge/staged-recipes/wiki/Frequently-asked-questions) for help.
```

Comment:
> How about noting that they should ignore/leave alone any other recipes in that folder?

Hoping we have made that easier to do. ( https://github.com/conda-forge/staged-recipes/pull/1373 ) 😉